src/watchful/client.py

```diff
@@ -400,21 +400,23 @@ def list_projects() -> List[Dict[str, Union[str, bool]]]:
     return json.loads(response.text)
 
 
-def open_project(id_: str) -> str:
+def open_project(id_: str, path: str) -> str:
```

Comment:
This raises alarms, for me. I'm not sure we want to expose a path like this. What happens when the project isn't stored on the filesystem anymore?

Reply:
the reality is that what we were calling `id_` was always the path, so really all I've done is split it up here. It was never not exposed...

> What happens when the project isn't stored on the filesystem anymore?
Caer and I were just discussing what it would look like for projects to be stored differently, we did however only discuss a different filesystem structure, not a "let's not use the filesystem" idea.